programmers/20210503/문자열-내-p와-y의-개수-세기.js

```diff
@@ -0,0 +1,13 @@
+function solution(s){
+    var count = 0;
+    for(var i = 0; i < s.length;  i++)
+        if (s[i] == "p" || s[i] == "P"){
```

Comment:
'==' 와 '==='의 차이를 알고 써야됨.
'=='은 값을 비교하는 것
'==='은 값과 자료형까지 비교하는 것
자바스크립트 에서는 '===' 쓰는 것이 좋음 
```suggestion
        if (s[i] === "p" || s[i] === "P") {
```

Reply:
요런 방법도 있습니다. toLowerCase()는 소문자로 바꾸는 것입니다.
toUpperCase()는 대문자로 바꾸는 것입니다.
```suggestion
        if (s[i].toLowerCase() === "p") {
```